src/TulsiGenerator/Scripts/bazel_build.py

```diff
@@ -752,6 +758,20 @@ def _BuildBazelCommand(self, options):
 
     return (bazel_command, 0)
 
+  def _HydrateIndexStore(self):
```

Comment:
This method should check the Xcode version I think it will cause issues otherwise.